packages/mcp/README.md

```diff
@@ -40,9 +40,17 @@ Create or update your VSCode MCP configuration file at `.vscode/mcp.json` in you
 
 ### Tools
 
+#### Component Tools
+
 - `get-vibe-component-metadata`: Returns metadata for a specific Vibe component (or all components if no name is provided).
 - `list-vibe-public-components`: Get a list of all public @vibe/core & @vibe/core/next components names.
 
+#### Icon Tools
+
+- `list-vibe-icons`: Get a list of all available Vibe icons from the @vibe/icons package with their descriptions, categories, and tags.
+- `get-vibe-icon-details`: Get detailed information about a specific Vibe icon including description, tags, category, and usage examples.
+- `search-vibe-icons`: Search and filter Vibe icons by text query, category, or tags. Supports filtering by categories ('Basic', 'Platform', 'View') and limiting results.
```

Comment:
I'm not sure about the value of it? Agent would probably make an early call to `list-vibe-icons` and then call `get`